lib/ansible/modules/cloud/amazon/ec2_vpc_dhcp_options.py

```diff
@@ -233,7 +239,8 @@ def ensure_tags(vpc_conn, resource_id, tags, add_only, check_mode):
         latest_tags = get_resource_tags(vpc_conn, resource_id)
         return {'changed': True, 'tags': latest_tags}
     except EC2ResponseError as e:
-        module.fail_json(msg=get_error_message(e.args[2]))
+        module.fail_json(msg=to_native(e.args[2]))
```

Comment:
Looking quickly at other modules, it looks like this is the only thing that looks at the `args` attribute. I'd do something like this instead:

```python
module.fail_json(msg=to_native(e))
```

It looks like there's a mix of stringifying `e` and returning `e.message`.